3p/bodymovinanimation.js

```diff
@@ -23,28 +24,47 @@ import {parseJson} from '../src/json';
  * @param {function(!Object)} cb
  */
 
+let animationHandler;
+
 function getBodymovinAnimationSdk(global, cb) {
   loadScript(global, 'https://cdnjs.cloudflare.com/ajax/libs/bodymovin/4.13.0/bodymovin_light.min.js', function() {
     cb(global.bodymovin);
   });
 }
 
+function parseMessage(event) {
+  const eventMessage = parseJson(getData(event));
+  const action = eventMessage['action'];
+  if (animationHandler) {
```

Comment:
There is a comment below that if done, means animationHandler can never be null when you get messages.